### PR TITLE
Create testwhitespace +some

### PR DESCRIPTION
**Pull request by dla-kirito**
test whitespace1

### CODE REVIEW

File: testwhitespace [hunk 1]
```diff
@@ -1,8 +1,8 @@
-const path = require('path');
-const fs = require('fs');
-const webpack = require('webpack');
-const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
-const SpeedMeasurePlugin = require('speed-measure-webpack-plugin');
+  const path = require('path');
+const fs =   require('fs');
```
Comment (dla-kirito) [inline testwhitespace:2]: test

File: testwhitespace [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+const webpack = require('webpack');
```
Comment (dla-kirito) [inline testwhitespace:3]: comment in commit

File: testwhitespace [hunk 3]
```diff
@@ -16,7 +16,6 @@ module.exports = [
         target: 'node',
         devtool: 'cheap-module-source-map',
       entry: {
-            extension:'./src/extension.ts',
```
Comment (dla-kirito) [inline testwhitespace]: tested

File: testwhitespace [hunk 4]
```diff
@@ -0,0 +1,81 @@
+  const path = require('path');
+const fs =   require('fs');
+const webpack = require('webpack');
```
Comment (dla-kirito) [inline testwhitespace:3]:
```suggestion
const webpack = require('webpack');  
```

**Comment by dla-kirito** [comment]
This branch has no conflicts with the base branchThis branch has no conflicts with the base branchThis branch has no conflicts with the base branchThis branch has no conflicts with the base branch

File: testwhitespace [hunk 5]
```diff
@@ -0,0 +1,81 @@
+  const path = require('path');
+const fs =   require('fs');
+const webpack = require('webpack');  
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin')  ;
+const S  peedMeasurePlugin = require('speed-measure-webpack-plugin');
+
+const appDirectory = fs.realpathSync(process.cwd());
+const resolveApp = (relativePath) => path.resolve(appDirectory, relativePath);
```
Comment (yuexi2021) [inline testwhitespace:8]: test

**Pull request closed without merging**